tasks/db_deploy/db_deploy.py

```diff
@@ -75,8 +73,8 @@ def task(config: Dict[str, str], orca_buckets: List[str]) -> None:
     schema.
 
     Args:
-        config (Dict): Dictionary of connection information.
-        orca_buckets: List[str]): List of ORCA buckets needed to create partitioned tables.
+        config: Dictionary of connection information.
+        orca_buckets: List of ORCA buckets needed to create partitioned tables.
```

Comment:
why are we removing the type?
I think keeping that will be easier for someone to know what kind of value to expect for the arguments.

Reply:
It's already covered in the function definition. Having it duplicated here means there's now redundant docs that must be updated when the parent variable changes.
There are also formatting issues. See differences between `config (Dict)` and `orca_buckets: List[str])`